lib/fog/kubevirt/compute/compute.rb

```diff
@@ -113,6 +113,115 @@ def version
           end
         end
 
+        #
+        # This class contains functions performing unit calculations.
+        # Originally implemented for manageiq-providers-kubevirt project
+        #
+        class UnitConverter
```

Comment:
@shiramax we should reusing this class for fixing https://projects.theforeman.org/issues/26656